reports/handler.go

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Abstract Machines
+// SPDX-License-Identifier: Apache-2.0
+
+package reports
+
+import (
+	"context"
+	"time"
+)
+
+func (re *report) StartScheduler(ctx context.Context) error {
+	defer re.ticker.Stop()
+
+	for {
+		select {
+		case <-ctx.Done():
+			return ctx.Err()
+		case <-re.ticker.Tick():
+			due := time.Now().UTC()
+
+			pm := PageMeta{
+				Status:          EnabledStatus,
+				ScheduledBefore: &due,
+			}
+
+			reportConfigs, err := re.repo.ListReportsConfig(ctx, pm)
+			if err != nil {
+				return err
```

Comment:
Use error handling similar to RE - we need to keep the loop if error happens, we can't exit on every error that may happen.